qkernel/src/lib.rs

```diff
@@ -561,9 +561,29 @@ pub extern "C" fn rust_main(
         CreateTask(ControllerProcess as u64, ptr::null(), true);
     }
 
+    if id == 2 {
```

Comment:
If vcpu blocks, it should be reflected also [here](https://github.com/QuarkContainer/Quark/blob/df21d540cdc55c0439b2ea0d64b4b5de7de7d79b/qvisor/src/runc/runtime/vm.rs#L277).

Reply:
fixed